terraform/GETTING_STARTED.md

```diff
@@ -51,7 +51,7 @@ Run:
 
 ```
 tctl create terraform.yaml
-tctl auth sign --format=tls --user=terraform --out=tf --ttl=10h
+tctl auth sign --format=file --user=terraform --out=terraform-identity --ttl=10h
 ```
 
```

Comment:
Tip: Make sure the user can impersonate the new terraform role and user. 

```
spec:
  allow:
    impersonate:
      roles:
      - terraform
      users:
      - terraform
```

Reply:
I'm not sure I understand. Why does the Terraform user/role need to impersonate itself?

Reply:
> Why does the Terraform user/role need to impersonate itself?

I had to add it my role, `access`, meaning that the `access` role can impersonate the user. This is important for cloud users. As they'll want to impersonate as the user and not login and obtain the certs as that user.